src/Lykke.Service.Dash.Api/Startup.cs

```diff
@@ -1,211 +1,35 @@
 ﻿using System;
-using System.Threading.Tasks;
-using Autofac;
-using Autofac.Extensions.DependencyInjection;
-using AzureStorage.Tables;
-using Common.Log;
-using Lykke.Common.ApiLibrary.Middleware;
-using Lykke.Common.ApiLibrary.Swagger;
-using Lykke.Logs;
-using Lykke.Service.Dash.Api.Core.Services;
-using Lykke.Service.Dash.Api.Core.Settings;
-using Lykke.Service.Dash.Api.Modules;
-using Lykke.SettingsReader;
-using Lykke.SlackNotification.AzureQueue;
-using Lykke.Service.BlockchainApi.Contract;
 using Microsoft.AspNetCore.Builder;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Newtonsoft.Json.Converters;
-using Newtonsoft.Json.Serialization;
+using Lykke.Sdk;
+using Lykke.Service.Dash.Api.Settings;
 
 namespace Lykke.Service.Dash.Api
 {
     public class Startup
     {
-        public IHostingEnvironment Environment { get; }
-        public IContainer ApplicationContainer { get; private set; }
-        public IConfigurationRoot Configuration { get; }
-        public ILog Log { get; private set; }
-
-        public Startup(IHostingEnvironment env)
-        {
-            var builder = new ConfigurationBuilder()
-                .SetBasePath(env.ContentRootPath)
-                .AddEnvironmentVariables();
-            Configuration = builder.Build();
-
-            Environment = env;
-        }
-
         public IServiceProvider ConfigureServices(IServiceCollection services)
         {
-            try
+            return services.BuildServiceProvider<AppSettings>(options =>
             {
-                services
-                    .AddMvc()
-                    .AddJsonOptions(options =>
-                    {
-                        options.SerializerSettings.Converters.Add(new StringEnumConverter { CamelCaseText = true });
-                        options.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
-                    });
-
-                services.AddSwaggerGen(options =>
+                options.ApiTitle = "Dash.Api";
+                options.Logs = logs =>
                 {
-                    options.DefaultLykkeConfiguration("v1", "Dash.Api");
-                    options.DescribeAllEnumsAsStrings();
-                    options.DescribeStringEnumsInCamelCase();
-                });
-
-                var builder = new ContainerBuilder();
-                var appSettings = Configuration.LoadSettings<AppSettings>();
-                Log = CreateLogWithSlack(services, appSettings);
+                    logs.AzureTableName = "DashApiLog";
+                    logs.AzureTableConnectionStringResolver = settings => settings.DashApiService.Db.LogsConnString;
+                };
```

Comment:
Add BlockChainIntegration and BlockChainIntegrationImportantMessages slack channels with min log level = Information and Warning respectively.